src/chocolatey.resources/helpers/functions/Install-ChocolateyZipPackage.ps1

```diff
@@ -196,14 +199,14 @@ param(
 
   $chocTempDir = $env:TEMP
   $tempDir = Join-Path $chocTempDir "$($env:chocolateyPackageName)"
-  if ($env:chocolateyPackageVersion -ne $null) { $tempDir = Join-Path $tempDir "$($env:chocolateyPackageVersion)"; }
+  if ($null -ne $env:chocolateyPackageVersion) { $tempDir = Join-Path $tempDir "$($env:chocolateyPackageVersion)"; }

```

Comment:
@it-praktyk curious how this is a code smell. Also, the commit message is going to need quite a bit more to explain the change, explain the reasoning, etc. Feel free to start the message with a good summary, but then dive into all of the why in the git commit message.

Reply:
OK, you are right. I've reworded the commit message.